data-labeling/environments/data-labeling/environment.yaml

```
@@ -0,0 +1,12 @@
+image:
+  name: azureml/curated/data-labeling
+  os: linux
+  context:
+    dir: context
+    dockerfile: Dockerfile
+    template_files:
+    - Dockerfile
+    - conda_dependencies.yaml
+  publish:
+    location: mcr
+    visibility: unlisted
```

This is required for all new environments:
```suggestion
    visibility: public
```